sonic_data_client/virtual_db.go

```diff
@@ -253,7 +398,7 @@ func v2rEthPortQueStats(paths []string) ([]tablePath, error) {
 	if strings.HasSuffix(paths[KeyIdx], "*") { // queues on all Ethernet ports
 		for que, oid := range countersQueueNameMap {
 			// que is in format of "Internal_Ethernet:12"
-			names := strings.Split(que, separator)
+			names := strings.Split(que, ":")
```

Comment:
Why switch to hard-coded separator?

Reply:
You are right. I had thought the ":" here in queue's name is hard-coded and different from database separator. I will change it back.